assets/src/util/pointLiterals.ts

```diff
@@ -0,0 +1,16 @@
+import { LatLngLiteral } from "leaflet"
+import { ShapePoint } from "../schedule"
+
+export const latLngLiteralToShapePoint = (
+  latLngLiteral: LatLngLiteral
+): ShapePoint => {
+  const { lat, lng } = latLngLiteral
+  return { lat, lon: lng }
+}
+
+export const shapePointToLatLngLiteral = (
+  shapePoint: ShapePoint
+): LatLngLiteral => {
+  const { lat, lon } = shapePoint
+  return { lat, lng: lon }
+}
```

Comment:
suggestion(non-blocking):
```suggestion
export const latLngLiteralToShapePoint = (
  {lat, lng}: LatLngLiteral
): ShapePoint => ({ lat, lon: lng })

export const shapePointToLatLngLiteral = (
  {lat, lon}: ShapePoint
): LatLngLiteral => ({ lat, lng: lon })
```